### PR TITLE
Override now handles taller inputs

### PR DESCRIPTION
**Pull request by AliceMarshall**

### CODE REVIEW

File: override/core/block_render_svg.html [hunk 1]
```diff
@@ -166,8 +171,8 @@
       (-Blockly.BlockSvg.DISTANCE_45_OUTSIDE - 0.5) + ',' +
       (Blockly.BlockSvg.CORNER_RADIUS -
       Blockly.BlockSvg.DISTANCE_45_OUTSIDE);
-  
-  
+
```
Comment (paulmck21) [inline override/core/block_render_svg.html]: Why all the gaps?

File: override/core/block_render_svg.html [hunk 2]
```diff
@@ -939,40 +948,49 @@
         if (field.renderWidth) {
           cursorX -= Blockly.BlockSvg.SEP_SPACE_X;
         }
-      } else {
+    } else if (input.renderHeight >= Blockly.BlockSvg.MIN_LARGE_BLOCK_Y && t === 0) {
```
Comment (paulmck21) [inline override/core/block_render_svg.html]: tabbing and space before else (housekeeping)

File: override/core/block_render_svg.html [hunk 3]
```diff
@@ -939,40 +948,49 @@
...
+  Blockly.Block.prototype.getMinHeight = function (height) {
+      return height + (this.getPaddingY() * 2);
   };
-  
+
   Blockly.BlockSvg.prototype.getTabPathUp = function () {
       return 'V ' + (Blockly.BlockSvg.TAB_HEIGHT - 4 + this.getPaddingY()) + ' c -' + Blockly.BlockSvg.TAB_WIDTH + ',0 -' + Blockly.BlockSvg.TAB_WIDTH + ',-8 0,-8';
   };
 
+
```
Comment (paulmck21) [inline override/core/block_render_svg.html]: why though?

File: override/core/block_render_svg.html (hunk 1 above)
Comment (AliceMarshall) [reply override/core/block_render_svg.html]: No clue...atom?

**Pull request merged**